### PR TITLE
fix some typos in GP API documentation

### PR DESCRIPTION
**Pull request by wamserma**
Signed-off-by: Markus S. Wamser <markus.wamser@mixed-mode.de>

### CODE REVIEW

File: documentation/globalplatform_api.md [hunk 1]
```diff
@@ -20,26 +20,25 @@ The TEE Client API describes and defines how a client running in a rich
...
+Trusted Application. When the session has been established the client has
 opened up the communication channel towards the specified Trusted Application
 identified by the `UUID`. At this stage the client and the Trusted Application
 can start to exchange data.
 
 
 ### TEE Client API example / usage
-Below you will find the main functions as defined by GlobalPlatform and which
-are used in the communication between the client and the TEE.
+Below you will find the main functions as defined by GlobalPlatform and used in the communication between the client and the TEE.
```
Comment (jforissier) [inline documentation/globalplatform_api.md]: Newline after "used" preferrably

Comment (wamserma) [reply documentation/globalplatform_api.md]: Fixed.

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

Thanks!

**Comment by etienne-lms** [comment]
FYI, can close this P-R. Fixes already applied in [optee_docs.git](https://github.com/OP-TEE/optee_docs).

**Pull request closed without merging**